src/analysis/analytic/isolated_zeros.lean

```diff
@@ -14,19 +14,22 @@ import topology.algebra.infinite_sum
 # Principle of isolated zeros
 
 This file proves the fact that the zeros of a non-constant analytic function of one variable are
-isolated. It also introduces a little bit of API in the `has_fpower_series_at` namespace that
-is useful in this setup.
+isolated. It also introduces a little bit of API in the `has_fpower_series_at` namespace that is
+useful in this setup.
 
 ## Main results
 
 * `analytic_at.eventually_eq_zero_or_eventually_ne_zero` is the main statement that if a function is
   analytic at `z₀`, then either it is identically zero in a neighborhood of `z₀`, or it does not
   vanish in a punctured neighborhood of `z₀`.
+* `analytic_on.eq_on_of_preconnected_of_frequently_eq` is the identity theorem for analytic
+  functions: if an analytic function `f` on a connected open set `U` is zero on set with an
+  accumulation point in `U` then `f` is identically `0` on `U`.
```

Comment:
```suggestion
  functions: if an analytic function `f` on a connected open set `U` is zero on a set
  with an accumulation point in `U`, then `f` is identically `0` on `U`.
```

Reply:
Ah, I should actually rewrite the comment because in the end the fact that `U` is open is not really needed (because the set of points at which `f` is analytic is always open).